thirdeye/thirdeye-frontend/app/helpers/manage-alert-utils.js

```diff
@@ -212,16 +264,17 @@ export function buildAnomalyStats(alertEvalMetrics, mode) {
   }
 
   anomalyStats.forEach((stat) => {
-    let origData = alertEvalMetrics.evalData[stat.key];
+    let origData = alertEvalMetrics.current[stat.key];
     let newData = alertEvalMetrics.projected[stat.key];
-    stat.value = formatEvalMetric(origData);
-    stat.projected = formatEvalMetric(newData);
-    if (stat.units) {
-      stat.valueUnits = isNaN(origData) ? null : stat.units;
-      stat.projectedUnits = isNaN(newData) ? null : stat.units;
-      stat.showDirectionIcon = !isNaN(origData) && !isNaN(newData) && origData !== newData;
-      stat.direction = stat.showDirectionIcon && origData > newData ? 'bottom' : 'top';
-    }
+    let isPercentageMetric = stat.units === '%';
+    let isTotal = stat.key === 'totalAlerts';
+    stat.showProjected = mode === 'explore';
+    stat.value = isTotal ? origData : formatEvalMetric(origData, isPercentageMetric);
+    stat.projected = isTotal ? newData : formatEvalMetric(newData, isPercentageMetric);
+    stat.valueUnits = isFinite(origData) ? stat.units : null;
```

Comment:
Looks like a lot of this data massaging is for UI only values. If those stats are not used for the logic, perhaps it'll be cleaner to use template helpers instead. Let's discuss this offline. (non-blocker for now)

Reply:
TBD